applications_tests/lethe-particles/depreciated_parameters.output

```diff
@@ -0,0 +1,65 @@
+Warning in file <depreciated_parameters.prm>: You are using the deprecated spelling <gx> of the parameter <g>.
+
```

Comment:
The name of the file should be 'deprecated' instead of depreciated, no?

Reply:
Yes haha